samples/fabric/fabric-cli/scripts/claimAsset.sh

```diff
@@ -17,14 +17,17 @@ cd scripts
 USER_CERT_BASE64=$(node getAssetTransferViewAddress.js getusercert $2 alice)
```

Comment:
Do we need this script anymore?

Reply:
No, we don't, actually Initially I was trying to change with in this, so may be that's why these have come up in this PR.
I thought to keep it till my changes were finalized. Should I delete them?

Reply:
Yes, you can delete them. If we ever need to refer to them, we can look at the revision history.